src/exampleTypes/TermsStatsTable.js

```diff
@@ -1,12 +1,83 @@
+import _ from 'lodash/fp'
 import React from 'react'
 import { observer } from 'mobx-react'
 import { exampleTypes } from 'contexture-client'
 import injectTreeNode from '../utils/injectTreeNode'
-import ExpandableTable from '../layout/ExpandableTable'
+import ExpandableTable, { Column } from '../layout/ExpandableTable'
+import { applyDefaults, inferSchema } from '../utils/schema'
 
 let TermsStatsTable = injectTreeNode(
-  observer(({ node, ...props }) => (
-    <ExpandableTable {...props} data={node.context.terms} />
+  observer(({ infer, fields, node, criteria, tree, children, ...props }) => (
+    <div>
+      <div>
+        Filter:
+        <input
+          type="text"
+          value={node.filter}
+          onChange={async e => {
+            await tree.mutate(node.path, { filter: e.target.value })
+            // TODO: This seems like a contexture-client issue. We shouldn't need this line:
+            await tree.refresh(node.path)
+          }}
+        />
+      </div>
+      <ExpandableTable
+        {...{
+          ...props,
+          children: [
+            ...children,
+            <Column
+              label="Controls"
+              expand={{
+                display: (value, record) => (
+                  <div>
+                    <button
+                      onClick={async () => {
+                        // TODO: Could this be smaller?
+                        let schema = _.flow(
+                          _.merge(infer && inferSchema(node)),
```

Comment:
Why would you infer the schema on a terms stats table? Key, count, etc won't be in the schema since they're aggregates